apps/user-portal/package.json

```diff
@@ -92,8 +92,10 @@
         "@types/lodash": "^4.14.146",
         "@types/ua-parser-js": "^0.7.33",
         "@wso2is/authenticate": "^0.1.0",
+        "@wso2is/fo2m": "^0.1.0",
```

Comment:
Is there a reason for naming the `@wso2is/fo2m`? I suggest `@wso2is/forms`

Reply:
+1, @thivi please proceed with the suggestion

Reply:
How about the name of the component? I used fo2m as the file name because I used the same name for the component. And we have to have a unique name for our component because a generic name may cause conflicts with other components we use. For instance, Semantic also has a Form component and if we name our component Form, then we won't be able use Semantic's Form.

Here, the name forms may not cause an issue but I think it is always better to go with a unique name to avoid conflicts in the future (and avoid generic names like form and forms). If not for fo2m, we will have to come up with another unique name IMO.